File: cs50-6-pset-sentimental-mariomore/mario.py
# TODO
from cs50 import get_int

while True:
    # prompt user for height
    height = get_int("Height: ")
    # number between 1 and 8
    if height > 0 and height < 9:
        break
k = 1

# 2D loop
for i in range(height):
    for j in range(height + 2 + k):
        # create spaces between pyramids
        if j == height or j == height + 1:
            print(" ", end="")
        elif j + i >= height - 1:
            print("#", end="")
        else:
            print(" ", end="")
    k += 1
    print()

File: cs50-1-pset-marioless/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)

{
    // the variable i want from the user
    int n;
//do while loop to obtain a number between 1 & 8
    do
    {
    n= get_int("Height: ");
    }
    while (n<1 || n>8);

//create rows
    for(int i=0; i<n; i++)
   {
    // columns
      for (int j = 0; j < n; j++)
      {
        //create blocks and spaces
         if((i + j)>=(n - 1))

            printf("#");
         else

            printf(" ");
       }
        //move to the next row
        printf("\n");
    }
}